nat-lab/tests/utils/vm/windows_vm_util.py

```diff
@@ -52,16 +59,43 @@ async def _copy_binaries(
         return
 
     try:
-        await connection.create_process(["rmdir", "/s", "/q", VM_TCLI_DIR]).execute()
-    except ProcessExecError as exception:
-        if (
-            exception.stderr.find("The system cannot find the file specified") < 0
-            and exception.stderr.find("The system cannot find the path specified") < 0
-        ):
-            raise exception
+        await connection.create_process(
+            ["taskkill", "/IM", "python.exe", "/F"]
+        ).execute()
+    except:
```

Comment:
Would be nice to handle exact exceptions here that we expect and not silence them all